src/RollupAggregrid.js

```diff
@@ -272,6 +272,21 @@ Ext.define('Jarvus.aggregrid.RollupAggregrid', {
         me.ungroupSubRecords(staleRecords, false);
     },
 
+    onClick: function(ev, target) {
+        var me = this,
+            containerEl = me.el;
+
+        if ((target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) && target.up('.jarvus-aggregrid-subrow')) { // eslint-disable-line no-cond-assign
+            return me.onCellClick(
+                parseInt(target.up('.jarvus-aggregrid-subrow').getAttribute('data-subrow-id'), 10),
+                parseInt(target.getAttribute('data-column-id'), 10),
+                target,
+                ev
+            );
+        }
+
+        me.callParent(arguments);
```

Comment:
@nbey this override interrupts the potential return value of onClick by not returning the result of me.callParent().

If you were using visual studio code with ESLint installed you'd have seen a warning about this, ensure you're using the eslint extension